src/BaywatchOperation.php

```diff
@@ -241,4 +241,20 @@ public function enable_tide_ui_restriction() {
     // Enable tide_ui_restriction module.
     $this->baywatch_install_module('tide_ui_restriction');
   }
+
+  public function import_default_section_config() {
+    $configs = [
+      'key.key.section_io_password',
+      'purge.logger_channels',
+      'purge.plugins',
+      'purge_queuer_coretags.settings',
+      'section_purger.settings.8714ff77fc',
```

Comment:
Is this `8714ff77fc` going to change for each app?

Reply:
@nicksantamaria, I checked and this value is only used internally by the purge module - it is not sent to the Section API so it can remain the same.